core/domain/topic_jobs_one_off.py

```diff
@@ -92,3 +92,55 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class UpdateTopicThumbnailSizeOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job to update the thumbnail_size_in_bytes in topic models. """
+
+    _DELETED_KEY = 'topic_deleted'
+    _ERROR_KEY = 'update_error'
+    _SUCCESS_KEY = 'thumbnail_size_updated'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [topic_models.TopicModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        topic = topic_fetchers.get_topic_by_id(item.id)
+
+        try:
+            # We are not updating thumbnail_filename here, but using it call the
```

Comment:
Missing word in sentence (around "call"), please proofread.

Reply:
Done.